### ISSUE TITLE
Exact Parameters To Reproduce Numbers in Paper

### ISSUE THREAD

**Issue by agnusmaximus** [msobroza/compositional_code_learning#2]
Hi, I'm having trouble reproducing the baseline numbers in the ICLR Deep Compositional Coding paper. Could you share the exact hyperparameters to run lstm_sent.py to get the 87% score?

**Comment by tginart** [comment]
Hi all, having the same issue as @agnusmaximus. Could we get some hyperparams to make this work?

Thank you!

**Comment by msobroza** [comment]
Hi @agnusmaximus and @tginart,

I obtained the results of using the baseline word vectors and the 64x8 compositional coding reconstruction vectors. And the last one obtain a gain of 1% (same relative improvement than in the paper) in IMDB sentiment analysis compared to the baseline. However, my GloVe baseline does only 84% of score. The baseline was produced using a single LSTM layer with 150 hidden units optimized with Adam optimizer for 15 epochs with a fixed learning rate of 0.0001. I got the results only on the end of 15 epochs. I think that there are two things that could improve both results:

1) Use a small validation dataset and get model parameters only on lowest validation loss iteration
2) Exclude OOV word vectors from vocabulary instead of random initialize them

If you find a solution or suggestion to improve baseline results please share with us

Thank you for your commentaries
